odoo/addons/base/models/res_config.py

```diff
@@ -470,6 +483,7 @@ def get_values(self):
     @api.model
     def default_get(self, fields):
         IrDefault = self.env['ir.default']
+        SuIrConfigParameter = self.env['ir.config_parameter'].sudo()
```

Comment:
`SuIrConfigParameter` → `IrConfigParameter`
The `Su` prefix is not useful for understanding the code below, the `sudo()` is only a marginal technical necessity.